src/dppm_rest_api.cr

```diff
@@ -18,6 +15,20 @@ module DppmRestApi
     end
 
     Kemal.config.add_handler Actions.auth_handler
+
+    Actions.has_access = ->(context : HTTP::Server::Context, permission : Access) {
```

Comment:
If we're going to make this change, we should have tests specifically for this proc, and in `spec_helper`, we should do `Actions.has_access = ->(context : HTTP::Server::Context, permission : Access) { true }` and expect a status code of 200.

Reply:
How to do this? I've tried something in `actions_spec.cr`